packages/website/components/page-templates/item-list.js

```diff
@@ -64,32 +65,48 @@ const RowCell = styled.div`
 
 const ItemList = ({ data, type }) => {
   const getRows = () => {
+    if (data.children) {
+      return data.children.map(child => renderRow(child));
+    }
+
     const packagePages = pageInfo.packages.find(
       pkg => pkg.packageId === data.id,
     );
     return packagePages[type].map(item => renderRow(item));
   };
-  const getDocsList = () => (
-    <Page>
-      <Title>
-        {type === 'docs' ? 'Document Home Page' : 'Example Home Page'}
-      </Title>
-      <Section>
-        <Table
-          head={head}
-          rows={getRows()}
-          defaultSortKey="name"
-          defaultSortOrder="ASC"
-        />
-      </Section>
-    </Page>
-  );
+  const getDocsList = () => {
+    let title = titleCase(data.id);
+    if (data.packageName) {
+      title = type === 'docs' ? 'Document Home Page' : 'Example Home Page';
+    }
+
+    return (
+      <Page>
+        <Title>{title}</Title>
+        <Section>
+          <Table
+            head={head}
+            rows={getRows()}
+            defaultSortKey="name"
+            defaultSortOrder="ASC"
+          />
+        </Section>
+      </Page>
+    );
+  };
 
   return (
     <NavigationWrapper
-      navContent={() => (
-        <PackageNavContent packageId={data.id} packageName={data.packageName} />
-      )}
+      navContent={() =>
+        data.packageName ? (
```

Comment:
I'm wondering whether it makes sense for this template component to have this logic. If we add more pages that use an item list will this ternary grow?

Reply:
I think at that point it's worth creating separate templates, but for now I'd prefer to just repurpose the existing template